app_sidebar_collapsible.py

```diff
@@ -200,16 +200,18 @@ def update_store_trips(start_date, end_date):
     Input('url', 'search'),
 )
 def display_page(search):
-    try:
-        if auth_type == 'cognito':
+    if auth_type == 'cognito':
+        try:
```

Comment:
why did you need to change the scope of the `if` block?

Reply:
In the authentication process, if something is wrong (like the AWS Cognito credentials), we shouldn't show the home page, and that's why I've used `try` here.

Reply:
And you have now moved both `get_cognito_login_page` to the exception handling? Why?

Reply:
> Right, but before this change, you had
> 
> ```
> try:
>     if
> ```
> 
> now you have
> 
> ```
> if
>    try:
> ```
> 
> Both of them seem to have the same functionality, so just trying to understand the reason for the change.

These changes are to increase the code readability. Other parts also didn't need to be inside the exception handling.

Reply:
> And you have now moved both `get_cognito_login_page` to the exception handling? Why?

The second one is outside of exception handling.